<launch>
  <!-- sicktoolbox node  -->
  <node name="LIDAR" pkg="sicktoolbox_wrapper" type="sicklms">
    <param name="port" value="/dev/serial/by-id/usb-Prolific_Technology_Inc._USB-Serial_Controller-if00-port0" />
    <param name="baud" value="38400"/>
  </node>
  <!-- transforms node -->
  <node name="transforms" pkg="transforms" type="tf_broadcaster" />
</launch>